debug_module.tex

```diff
@@ -18,12 +18,15 @@ \chapter{Debug Module (DM)} \label{dm}
 \item Allow direct System Bus Access. (Optional)
 \end{steps}
 
-In order to implement memory access, a target must implement at least one of
-Program Buffer, System Bus Access, or Abstract Access Memory command
-mechanisms.
-
-In order to implement access to all registers, targets should implement either
-the Program Buffer or abstract access to all registers.
+\begin{steps}{In order to be compliant with this specification an
+    implementation must:}
+\item Implement all the required features listed above.
+\item Implement at least one of Program Buffer, System Bus Access, or Abstract
+    Access Memory command mechanisms.
+\item Implement either the Program Buffer, or abstract access to all registers
+    that are visible to software running on the hart including all the
+    registers that are present on the hart and listed in Table~\ref{tab:regno}.
```

Comment:
I'm approving this, though I also would approve @asb's suggestion of "Minimal compliance". Though that's a whole nother level of something to define, and I think adding the program buffer is always the right choice, so...